src/backend/distributed/citus--5.1-1--5.1-2.sql

```diff
@@ -0,0 +1,6 @@
+CREATE FUNCTION master_multi_shard_modify(text)
```

Comment:
I thought about that before changing the name and liked it. It is short and easy to write. However, master_modify_shards name not sounded intuitive. It was like a function that "modifies something about the shards". 

Since, before accepting UPDATE, the name was master_delete_from_shards, another alternative can be "master_modify_on_shards". It is intuitive to me but ugly.

I am OK if "master_modify_shards" is your preference.